src/components/navigationbar.scss

```diff
@@ -65,3 +66,25 @@
     }
   }
 }
+
+@media #{$mobile-version} {
```

Comment:
question: I noticed that in both changes you did not use "mobile first approach". Why?

Reply:
From what I understand from the article you sent me, choosing the destock design or the mobile-first approach is something you do at the very beginning. Now that I did 95% of the job, I won't do rebuild the code from scratch. I'll do it for another project.